package.json

```diff
@@ -190,6 +190,24 @@
                 "title": "Clear",
                 "category": "Notebook",
                 "icon": "$(clear-all)"
+            },
+            {
+                "command": "jupyter-kernelManager.viewKernelMessagesAsTree",
+                "title": "Tree View",
+                "category": "Notebook",
+                "icon": "$(list-tree)"
+            },
+            {
+                "command": "jupyter-kernelManager.viewKernelMessagesAsList",
```

Comment:
Viewing the messages as a list view was important when working on IW debugger & IPyWidgets (to see the sequence of messages as they were sent/received)